dev-java/rxtx/rxtx-2.2_pre2-r2.ebuild

```diff
@@ -18,14 +18,21 @@ SLOT="2"
 KEYWORDS="amd64 x86"
 IUSE="lfd"
 
-RDEPEND=">=virtual/jre-1.6
+RDEPEND=">=virtual/jre-1.8:*
 	lfd? ( sys-apps/xinetd )"
 
-DEPEND=">=virtual/jdk-1.6
+DEPEND=">=virtual/jdk-1.8:*
 	app-arch/unzip"
```

Comment:
again here, `app-arch/unzip` should go to `BDEPEND` as it's here to unpack the .zip file. `BDEPEND` was introduced in eapi 7 iirc so in older versions of eapi it had no better place than `DEPEND` but in eapi 7 the right place is `BDEPEND`.

Reply:
fixed.